src/utils/isModelValueType.ts

```diff
@@ -0,0 +1,11 @@
+import { ModelValueType } from '../glossary'
+
+export function isModelValueType(value: any): value is ModelValueType {
```

Comment:
What is the benefit of abstracting this? Do we use this check often?

Reply:
Okay, I see. We're reusing it in `createModel` within the nullable property check.